ui/appui-layout-react/src/test/state/NineZoneStateReducer.test.ts

```diff
@@ -1619,6 +1620,22 @@ describe("NineZoneStateReducer", () => {
         widgetIndex: 0,
       });
     });
+
+    it("should popout a tab and fit to content container", () => {
+      const blankHTML = document.createElement("div");
+
+      stub(document, "getElementById").returns(blankHTML);
+
+      let state = createNineZoneState();
+      state = addTab(state, "t1");
+      state = addPanelWidget(state, "left", "w1", ["t1"]);
+
+      const newState = NineZoneStateReducer(state, {
+        type: "WIDGET_TAB_POPOUT",
+        id: "t1",
+      });
+      newState.popoutWidgets.allIds.should.length(1);
```

Comment:
Shouldn't we check the size ? How is this validating that we are indeed fitting to the content container ?

Reply:
It was just checking that it successful created the popout widget using the content container as the bounds. Added in the size check for the blank div + the buffer space